### PR TITLE
add ntpd_configure_restrictions for rhel7

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

A rule ntpd_configure_restrictions is added for rhel7. Also additional CPE for rhel7 is added called ntp and it checks if ntp package is installed. This is done because the mentioned rule is applicable only if ntp is installed.

#### Rationale:

The rule is part of CIS rhel7 benchmark.

### CODE REVIEW

File: rhel7/cpe/rhel7-cpe-dictionary.xml [hunk 1]
```diff
@@ -62,6 +62,11 @@
             <!-- the check references an OVAL file that contains an inventory definition -->
             <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="filename">installed_env_has_nss-pam-ldapd_package</check>
       </cpe-item>
+      <cpe-item name="cpe:/a:ntp">
+            <title xml:lang="en-us">Package ntp is installed</title>
+            <!-- the check references an OVAL file that contains an inventory definition -->
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="filename">installed_env_has_ntp_package</check>
+      </cpe-item>
```
Comment (yuumasato) [inline rhel7/cpe/rhel7-cpe-dictionary.xml]: As the rule is applicable to Fedora, this same entry needs to be added to Fedora CPE dictionary too.

**Comment by vojtapolasek** [comment]
Thank you, fixed and rebased on master.

**Comment by jan-cerny** [comment]
@vojtapolasek you still have conflicts

@matejak ping

File: linux_os/guide/services/ntp/ntpd_configure_restrictions/rule.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+documentation_complete: true
+
+prodtype: rhel7,fedora
+
+title: 'Configure server restrictions for ntpd'
+
+description: |-
+    ntp is a daemon which implements the Network Time Protocol (NTP). It is designed to
```
Comment (jan-cerny) [inline linux_os/guide/services/ntp/ntpd_configure_restrictions/rule.yml]:
```suggestion
    ntpd is a daemon which implements the Network Time Protocol (NTP). It is designed to
```

File: linux_os/guide/services/ntp/ntpd_configure_restrictions/rule.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    synchronize system clocks across a variety of systems and use a source that is highly
+    accurate. More information on NTP can be found at
+    {{{ weblink(link="http://www.ntp.org")}}}.
+    ntp can be configured to be a client and/or a server.
+    To ensure that ntpd implements correctt server restrictions, make sure that the following lines exist in the file <tt>/etc/ntpd.conf</tt>:
```
Comment (jan-cerny) [inline linux_os/guide/services/ntp/ntpd_configure_restrictions/rule.yml]:
```suggestion
    To ensure that ntpd implements correct server restrictions, make sure that the following lines exist in the file <tt>/etc/ntpd.conf</tt>:
```

**Pull request merged**